lib/confluence.js

```diff
@@ -226,7 +226,7 @@ Confluence.prototype.postContent = function(space, title, content, parentId, cal
  * @param {string} content
  * @param {Function} callback
  */
-Confluence.prototype.putContent = function(space, id, version, title, content, callback){
+Confluence.prototype.putContent = function(space, id, version, title, content, callback, minorEdit = false){
```

Comment:
@arthmoeros - I plan to edit this a bit to ensure old versions of node are supported. I will remove the default assignment and change line 239 to be `"minorEdit": minorEdit || false`.  Let me know if you have flags. Thanks!

Reply:
No problem, go ahead.